lib/tasks/config.rake

```diff
@@ -2,7 +2,7 @@ require 'optparse'
 require 'json'
 require 'yaml'
 
-desc "Configure Foreman in-database settings, see rake config -- --help for more details"
+desc "Configure Foreman in-database settings, see rake -- config --help for more details"
```

Comment:
Feel free to change it to `foreman-rake config --help`, I don't think it's that much clearer as it is right now.